Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/ruleset/RulesetManagement.java

```diff
@@ -65,6 +68,32 @@
         return acquisitionStageNames;
     }
 
+    @Override
+    public List<String> getIdsOfKeysForSpecialField(SpecialField specialField) {
+        return getIdsOfKeysForSpecialField(ruleset.getKeys(), specialField);
+    }
+
+    private List<String> getIdsOfKeysForSpecialField(List<Key> keys, SpecialField specialField) {
+        ArrayList<String> idsOfKeysForSpecialField = new ArrayList<>(1);
+        for (Key key : keys) {
+            if (key.getKeys().isEmpty()) {
+                if (Objects.isNull(key.getUse())) {
+                    continue;
+                }
+                Set<SpecialField> uses = SpecialField.valuesOf(key.getUse());
+                if (uses.contains(specialField)) {
+                    idsOfKeysForSpecialField.add(key.getId());
+                }
+            } else {
+                List<String> idsOfKeysOfKey = getIdsOfKeysForSpecialField(key.getKeys(), specialField);
+                for (String idOfKeyOfKey : idsOfKeysOfKey) {
+                    idsOfKeysForSpecialField.add(key.getId() + '/' + idOfKeyOfKey);
```

Comment:
Is this a convention? Where do I know, that a slash separates a key from the keyOfKey

Reply:
I will change this to `@`, then it is identical to the spelling in the Metadata copy rules.